<launch>

    <machine
        name="robot"
        address="ddzz"
        user="ubuntu"
        env-loader="/home/ubuntu/setup_ros.bash"
        default="true"
    />
    
    <node pkg="ddzz_run" type="find_usb_devices.py" name="find_usb_devices"/>

</launch>